scvi/model/base/_vaemixin.py

```diff
@@ -14,6 +16,7 @@ class VAEMixin:
     """Univseral VAE methods."""
 
     @torch.no_grad()
+    @unsupported_in_latent_mode
```

Comment:
Because latent mode doesn't necessarily mean the count data has been set to 0, this would technically be supported (especially with the other changes mentioned). However, if the `memory_reduce` option has been used, the counts will be 0, so the ELBO will be nonsensical. Therefore the decorator can instead add a warning that memory reduce has been used so the output of this function might not make sense.